### PR TITLE
feat: `create_evm_vk` python bindings 

### PR DESCRIPTION
**Pull request by ethan-crypto**

### CODE REVIEW

File: src/python.rs [hunk 1]
```diff
@@ -1533,6 +1533,56 @@ fn create_evm_verifier(
...
+/// Arguments
+/// ---------
+/// vk_path: str
+///     The path to the verification key file
+///
+/// settings_path: str
+///     The path to the settings file
+///
+/// sol_code_path: str
+///     The path to the create the solidity VK
```
Comment (JSeam2) [inline src/python.rs:1547]: This docstring is probably gonna confuse people. Solidity VK won’t make sense to anyone else.

File: src/python.rs [hunk 2]
```diff
@@ -1533,6 +1533,56 @@ fn create_evm_verifier(
     })
 }
 
+/// Creates an Evm VK
```
Comment (JSeam2) [inline src/python.rs:1536]: Would be good to eli5 about this because this will make 0 sense to an outsider

**Review by JSeam2 — APPROVED** [review]

**Review by alexander-camuto — APPROVED** [review]
